data-processing/common/commands/compute_iou.py

```diff
@@ -87,69 +121,112 @@ def load(self) -> Iterator[IouJob]:
             output_root = directories.arxiv_subdir("bounding-box-accuracies", arxiv_id)
             file_utils.clean_directory(output_root)
 
-            actual: Dict[
-                Tuple[PageNumber, EntityType], List[FrozenSet[FloatRectangle]]
-            ] = {}
-
-            entity_types = [
-                EntityKeyPair("citations", "citation"),
-                EntityKeyPair("equations", "equation"),
-                EntityKeyPair("equation-tokens", "symbol"),
+            entity_keys = [
+                # EntityKeys("citations", "citation"),
+                # EntityKeys("equations", "equation"),
+                EntityKeys("symbols", "symbol"),
```

Comment:
what's happening here w/ the commenting-out?

Reply:
I am updating the script to only compute accuracy of symbol detection, and not the accuracy of citation and equation detection. Though I should revert this to remove the comments before merge.